concent-builder/configure.yml

```diff
@@ -64,3 +81,32 @@
             state: directory
             owner: "{{ shared_user }}"
             group: "{{ shared_user }}"
+
+        - name:    Create jenkins home directory
+          file:
+            path:  "{{ jenkins_home_dir }}/"
+            state: directory
+            owner: jenkins
+            group: jenkins
+
+        - name:   Add jenkins role to sudoers
+          copy:
+            src:  "files/jenkins/jenkins_sudoers"
+            dest: "/etc/sudoers.d/jenkins"
```

Comment:
This is no longer necessary. I see that you have changed the code so that jenkins no longer needs to do anything as root.